XP-Pen_G430/index.md

```diff
@@ -0,0 +1,19 @@
+---
+VID: "28bd"
+PID: "0075"
+vendor: Ugtablet
+product: Tablet G3 4x3
```

Comment:
Once you've determined the name of a tablet, you need to use it everywhere: in directory name (transformed into an "identifier"), in the `index.md` (in `vendor` and `product`), and in photo name (again as an "identifier"), if the photo is of the same brand/model.

So the above `vendor` and `product` should match the directory name. I.e. in this case they should be `XP-Pen` and `G430`.

Reply:
I'll need to put comments into the schema file to describe the fields better.